pkg/cli/cliflags/flags.go

```diff
@@ -488,17 +488,31 @@ a public network without combining it with --listen-addr.`,
 The path to the directory containing SSL certificates and keys.
 <PRE>
 
-Cockroach looks for certificates and keys inside the directory using the
+Cockroach looks for required certificates and keys inside the directory using the
 following naming scheme:
 
-  - CA certificate and key: ca.crt, ca.key
-  - Server certificate and key: node.crt, node.key
-  - Client certificate and key: client.<user>.crt, client.<user>.key
+  - ca.crt:                               CA certificate. Used to verify all certificates.
+  - node.crt, node.key:                   Server certificate and key
+  - client.<user>.crt, client.<user>.key: Client certificate and key
+
+All certificates must be signed by ca.crt.
```

Comment:
Suggestion: "Server and client certificates must be signed by ca.crt"

Reply:
Done